src/Layout/Nav/Nav.scss

```
@@ -50,6 +50,16 @@
       background-size: contain;
       background-position: center bottom;
       background-repeat: no-repeat;
+      position: relative;
+    }
+
+    .alert {
+      position: absolute;
+      top: 50%;
+      left: 100%;
+      font-size: 1.5rem;
+      font-weight: bold;
+      color: $color-accent;
     }
```

just style - display in circle 
```suggestion
    .alert {
      position: absolute;
      right: -10px;
      top: -5px;
      background-color: $color-accent;
      width: 20px;
      height: 20px;
      border-radius: 50%;
      font-size: 1.5rem;
      font-weight: bold;
      color: $color-primary;
    }
```